docs/contributor.md

```diff
@@ -121,13 +42,15 @@ y2038.ctl
 und y2038.xml  
 
 Each file in the scripts folder has a corresponding result file in the result files dir: same name, different extension
-Subdirs are allowed to organize testases, must appera in bith directories; simply create new directories and throw your testcode, resp result files in there.  
+Sub directories are allowed to organize test cases, must appear in both directories; simply create new directories and throw your test-code, resp result files in there.  
 
 Errors will be thrown, if for a .ctl file no .xml can be found.  
 If there is an error is in the .ctl file and no corresponding <error> in the .xml file the test will pass.  
 If no error is in the .ctl file and an error is in the .xml file a warning will be issued.  
 
-Testresults will be written to: "<project_dir>/log/CtrlppCheck/testscripts/suspiciousSemicolon.xml"
+Test results will be written to: "<project_dir>/log/CtrlppCheck/testscripts/suspiciousSemicolon.xml"
+
+Please keep in mind, that good documented test is the best example. Therefor document the test-script as well, that everybody can see and understand, what the check does and what is the (un)expected result. Also when you find some limitation, it is still good idea to document it in the test-code. It is public and transparent. Also the next developer will find possible leaks faster.
```

Comment:
```suggestion
Please mind that well documented tests are the best examples. Therefore, document the test-script as well so that everybody can see and understand what the check does and what the (un)expected results are. Also if you find limitations, it is good practice to document them in the test-code as well. This is transparent for the public community and makes it easier for others to steps in and proceed.
```